config/clusters/maap/prod.values.yaml

```diff
@@ -5,45 +5,236 @@ userServiceAccount:
   annotations:
     eks.amazonaws.com/role-arn: arn:aws:iam::916098889494:role/maap-prod
 jupyterhub:
+  custom:
+    2i2c:
+      add_staff_user_ids_to_admin_users: false
+    homepage:
+      gitRepoBranch: master
+      gitRepoUrl: https://github.com/MAAP-Project/maap-hub-homepage
   singleuser:
     extraEnv:
       SCRATCH_BUCKET: s3://maap-scratch-prod/$(JUPYTERHUB_USER)
+      MAAP_API_HOST: api.maap-project.org
+      DOCKERIMAGE_PATH_DEFAULT: mas.maap-project.org/root/maap-workspaces/custom_images/maap_base:v5.0.0
```

Comment:
This will set this as the environment variable no matter what image is used. Is that what was expected? In the last PR, I saw this was set to be the same as the name of the image, in which case it should use `$(JUPYTER_IMAGE)` as the value.